zeppelin-zengine/src/test/java/org/apache/zeppelin/conf/ZeppelinConfigurationTest.java

```diff
@@ -69,18 +69,18 @@ public void getAllowedOriginsNoneTest() throws MalformedURLException {
   }
 
   @Test
-  public void isWindowsPathTestTrue() {
+  public void checkWindowsTestTrue() {
 
     ZeppelinConfiguration conf = ZeppelinConfiguration.create("zeppelin-test-site.xml");
-    Boolean isIt = conf.isWindowsPath("c:\\test\\file.txt");
+    Boolean isIt = conf.checkWindows();
     Assert.assertTrue(isIt);
   }
 
   @Test
-  public void isWindowsPathTestFalse() {
+  public void checkWindowsTestFalse() {
 
     ZeppelinConfiguration conf = ZeppelinConfiguration.create("zeppelin-test-site.xml");
-    Boolean isIt = conf.isWindowsPath("~/test/file.xml");
+    Boolean isIt = conf.checkWindows();
     Assert.assertFalse(isIt);
   }
```

Comment:
How do the tests work?

Reply:
I will question it.

I only changed the function name in the previous version for the time being. The internal code is the same as before.

_SystemUtils.IS_OS_WINDOWS_ will have different results for each operating system, so how can I create test code?

Do you have any good ideas?

Thank you.

Reply:
Previously written tests were checked using file paths, but as this function becomes system-dependent, the meaning of the previous tests seems to have changed.
If you write a test, it would be better to mock `checkWindows` and test the result value of a function that uses `checkWindows`, such as `getAbsoluteDir` and `getKeyStorePath` .

Reply:
We can define a new CI env like `CI_LINUX=TRUE` in Github action and use it when testing like
```
    if(System.getenv("CI_LINUX").equals("TRUE")) {
      // Github action
      Assert.assertFalse(conf.checkWindows());
    } else if (System.getenv("CI_WINDOW").equals("TRUE")) {
      // Appveyor
      Assert.assertTrue(conf.checkWindows());
    } else {
      // Local
      Assert.assertFalse(conf.checkWindows());
    }
```